tools/deployment-cli-tools/ch_cli_tools/utils.py

```diff
@@ -424,6 +424,29 @@ def check_docker_manifest_exists(registry, image_name, tag, registry_secret=None
     resp = requests.get(api_url)
     return resp.status_code == 200
 
+
+def filter_empty_strings(value):
+    return value != ""
+
+
+def search_word_in_file(file, word):
```

Comment:
suggestion: filename would be a better name here as the caller would expect to pass a file object here. Also using type hints would help